config-coral.ini

```diff
@@ -119,7 +118,8 @@ Name = file_system_logger
 TimeInterval = 0.5
 LogDirectory = /repo/data/processor/static/data/sources
 ; Screenshot time is measured in minutes (if period <= 0 then no screenshots are uploaded)
-ScreenshotPeriod = 0
+ScreenshotPeriod = 5
```

Comment:
Do we want to enable this by default?

Reply:
Yes, the new dashboard will include a section with a "live" screenshot of each camera.